pkg/apis/extensions/install/install.go

```diff
@@ -56,6 +54,15 @@ func init() {
 		return
 	}
 
+	registered.RegisterVersions(availableVersions...)
+
+	if err := registered.EnableVersions(externalVersions...); err != nil {
+		glog.V(4).Infof("%v", err)
+		return
+	}
+
+	addVersionsToScheme(externalVersions...)
```

Comment:
line 64 to line 82 here is the code that really "enables" the versions. I guess we can move that into an enableVersions() method and call it here.